tests/test_core_schedule.py

```diff
@@ -132,11 +132,12 @@ def test_initiating_schedule(schedule):
                                'crs': 'epsg:27700'})
 
 
-def test_initiating_schedule_with_non_uniquely_indexed_objects():
+@pytest.fixture()
+def non_uniquely_indexed_schedule():
     route_1 = Route(route_short_name='name',
                     mode='bus', id='',
-                    stops=[Stop(id='1', x=4, y=2, epsg='epsg:27700'), Stop(id='2', x=1, y=2, epsg='epsg:27700'),
-                           Stop(id='3', x=3, y=3, epsg='epsg:27700'), Stop(id='4', x=7, y=5, epsg='epsg:27700')],
+                    stops=[Stop(id='1', x=4, y=2, epsg='epsg:4326'), Stop(id='2', x=1, y=2, epsg='epsg:4326'),
```

Comment:
I'm glad you picked up on this, because I made a mistake/had a full blown brain fart. I misread the name/started using the word 'indexing' for 'projection' (I think) and ended up with this monstrocity. I tidied it up